src/inc/regdisp.h

```diff
@@ -8,7 +8,7 @@
 
 #ifdef DEBUG_REGDISPLAY
 class Thread;
-
+struct REGDISPLAY;
```

Comment:
Since Arm64 also defines WIN64EXCEPTIONS like x64, I think we should understand how it gets declared for x64 and see if we can unify the declaration for our 64bit platforms (or for WIN64EXCEPTIONS based approach) instead of explicitly declaring it here.